### ISSUE TITLE
Create docs page about pair programming program

### ISSUE THREAD

**Issue by kkemple** [gatsbyjs/gatsby#7082]
## Summary

Currently it's not documented anywhere that people can sign up to get help with Gatsby projects from core team members. It would also be nice to show a version of the calendar so people could see it's being used. We need to be careful not to display any personal info about participants unless given express permission.

### Motivation

Getting the word out about this program so that everyone gets a chance to participate! 📣

**Comment by zauni** [comment]
Do you mean the link to calendly should be in the docs?
https://calendly.com/gatsbyjs/pair-programming

**Comment by kkemple** [comment]
Yeah! As well as what the process is like, etc! This is still being figured out though tbh :smile:
